build_shared.gradle

```diff
@@ -12,7 +12,7 @@ repositories {
     mavenCentral()
     maven gradleutils.forgeMaven
     maven gradleutils.minecraftLibsMaven
-    //mavenLocal()
+    mavenLocal()
```

Comment:
Make sure you undo this, because this can lead to using libraries from maven local that don't match what others can pull.

Reply:
I'm aware. I'm still going to treat it like a PR so that I can learn how this magic works by doing my usual code review stuff.